lib/hobbes/ipc/prepl.C

```diff
@@ -208,15 +209,27 @@ void runMachineREPLStep(cc* c) {
 
       dbglog("eval '" + expr + "'");
 
+      // buffer the result to remove any accidental internal terminators
+      std::ostringstream ss;
+      auto* stdoutbuffer = std::cout.rdbuf();
```

Comment:
I think simply using `auto` is good enough?

Reply:
maybe we could combine two lines into one: `auto stdoutbuffer = std::cout.rdbuf(ss.rdbuf());`